spec/libsass-todo-issues/issue_1098/expected.expanded.css

```diff
@@ -0,0 +1,5 @@
+div {
+  opacity: 1\9;
+  width: 500px\9;
+  color: #f00\9\0\;;
```

Comment:
IMO that looks correct as one `;` is escaped, as in the input. IMO the delimiter is optional for the last declaration!